packages/unified-latex-util-arguments/libs/gobble-single-argument.ts

```diff
@@ -188,6 +171,31 @@ export function gobbleSingleArgument(
             }
             break;
         }
+        case "embellishment": {
+            argument = [];
+            // Split tokens into single characters
+            const tokens = argSpec.embellishmentTokens.flatMap(token => {
+                if (typeof token === 'string') return token.split('');
+                console.warn(
+                    `Embellishment token should be a single character, but got ${stringifyGroup(token)}`
+                );
+                return [];
+            });
+            for (let token of tokens) {
```

Comment:
Use `const` if you're not mutating the value.

Reply:
Fixed.